app/models/organization/creator.rb

```diff
@@ -66,12 +66,18 @@ def perform
       begin
         github_organization = GitHubOrganization.new(users.first.github_client, github_id)
 
+        organization_webhook = OrganizationWebhook.find_or_initialize_by(github_organization_id: github_id)
+        webhook_id = create_organization_webhook!
```

Comment:
I think it makes sense to move the logic for creating a webhook from the `Organization::Creator` to this new `OrganizationWebhook` model so that we don't have the code in multiple places.

@BenEmdon What do you think? I can move these methods over in a future PR if you'd rather focus on what you're working on

Reply:
Totally agree 👍 That work aligns with the effort I am making in #1647 and I'll get to that in a follow up PR 😄